pytest_fixtures/broker.py

```diff
@@ -83,6 +83,13 @@ def rhel8_contenthost():
         yield host
 
 
+@pytest.fixture(scope="module")
+def rhel8_contenthost_module():
+    """A fixture that provides a content host object based on the rhel8 nick"""
+    with VMBroker(nick='rhel8', host_classes={'host': ContentHost}) as host:
```

Comment:
Is there a reason to use RHEL8 as oppose to RHEL7? Asking because there's no rhel8 template to use as far as I'm aware.

Reply:
I take that back

Reply:
I choose rhel8 because:
To test the remediation feature properly I need a misconfigured host. Meaning I need host which have insights recommendation which can be remediated from Satellite. There's a way for me to break rhel8 host and get 
recommendation for it. By removing `best` option from `/etc/dnf/dnf.conf` file I will get insights recommendation to fix it(i.e `The dnf installs lower versions of packages when the "best" option is not present in the /etc/dnf/dnf.conf`). And because I know what the recommendation is then I can use that to verify if satellite remediated the issue or not.

Reply:
@latran we do have rhel8 templates and nick available internally.